styleguide/README.md

```diff
@@ -624,6 +624,24 @@ gem "redis", "~> 2.0"
 ---
 ### Active Record
 
+#### Prefer non-dynamic dirty tracking methods.
+
+Do e.g.
+
+``` ruby
+attribute_was("title")
```

Comment:
This one reads a lot worse, like some attribute had the value "title"

Reply:
Yup, like I said, the docs seem to use the dynamic ones. So I'm sure that's DHH's preference and probably the most common style. But it's a public method, so it's not considered internal.

You're right that `attribute_was("title")` doesn't read great. Looks like it might be gone in 6.0, replaced with `attribute_before_last_save` and `attribute_in_database`. I'll update the example!

Reply:
Ah, not gone (you linked to its docs), but moved out of https://apidock.com/rails/ActiveRecord/AttributeMethods/Dirty into ActiveModel.